src/get_functions.jl

```diff
@@ -0,0 +1,19 @@
+
+"""
+get_origin(vtk_file)
```

Comment:
```suggestion
    get_origin(vtk_file)
```
To get correct rendering as code